src/components/victory-gauge.js

```diff
@@ -302,75 +321,176 @@ export default class VictoryGauge extends React.Component {
       .sort(null)
       .startAngle(degreesToRadians(props.startAngle))
       .endAngle(degreesToRadians(props.endAngle))
-      .padAngle(degreesToRadians(props.padAngle))
-      .value((datum) => { return datum.y; });
+      .padAngle(degreesToRadians(props.padAngle));
+  }
+  getSegments(props) {
+    const {segments} = props;
+    if (segments && segments.length) {
+      return segments;
+    }
+    // if there are no segments provide set of data as a
+    // segment that spans entire arc. array of 1?
+    return [1];
+  }
+  getGaugeRange(props, segmentLocations) {
+    const radiansToDegrees = (r) => r * (180 / Math.PI);
+    const {domain} = props;
+    return {
+      minimum: {
+        value: domain && domain[0] || segmentLocations[0].data,
+        degrees: radiansToDegrees(segmentLocations[0].startAngle)
+      },
+      maximum: {
+        value: domain && domain[1] || segmentLocations[1].data,
+        degrees: radiansToDegrees(segmentLocations.reverse()[0].endAngle)
```

Comment:
I see what you're saying. I think I led you astray with my comparison between segments and pie data. Now that I'm looking at your demo having this
```
<VictoryGauge
     data={32}
      segments={[1, 2, 3]}
       tickValues={[10, 20, 30, 60]}
          />
```
create a gauge that looks like this
![screen shot 2016-05-26 at 4 08 22 pm](https://cloud.githubusercontent.com/assets/3719995/15592873/2cfbdf22-235c-11e6-9d6c-7f715d11a06b.png)

is really confusing and incorrect. The segments should be referring to the same type of value as the tickValues